<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>298</width>
      <height>227</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progress1">
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress2">
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>100</number>
       </property>
       <item>
        <widget class="QPushButton" name="switch1">
         <property name="text">
          <string>Switch player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="switch2">
         <property name="text">
          <string>Switch player</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gameState">
         <property name="text">
          <string>Select playtime and press start game!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="shortGame">
         <property name="text">
          <string>120 sec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="longGame">
         <property name="text">
          <string>300 sec</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="start">
         <property name="text">
          <string>START GAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="text">
          <string>STOP GAME</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
